react/features/base/util/uri.js

```diff
@@ -556,6 +556,20 @@ export function urlObjectToString(o: Object): ?string {
         }
     }
 
+    const { brandLogoPath } = o;
```

Comment:
Why did you assign brandLogoPath to o?

Reply:
Here I did `o` object destructuring and assign the value from external api to enhance url.